app/views/admin/loans/content/_breakeven_charts.html.slim

```diff
@@ -1,20 +1,20 @@
-.summary-chart.breakeven-chart.row
-  h3 = t("loan.breakeven.charts_title")
-  .chart.col-lg-6
-    h4 = t("loan.breakeven.revenue_by_product")
-    #breakeven-revenue-chart
-  .chart.col-lg-6
-    h4 = t("loan.breakeven.production_cost_by_product")
-    #breakeven-production-cost-chart
-  .chart.col-lg-6
-    h4 = t("loan.breakeven.profit_by_product")
-    #breakeven-product-profit
-  .chart.col-lg-6
-    h4 = t("loan.breakeven.fixed_costs", count: 2)
-    #breakeven-fixed-cost-chart
-  .chart.col-lg-6
-    h4 = t('loan.breakeven.total_cost_breakdown')
-    #breakeven-costs-chart
+.summary-chart.breakeven-chart
+  h3.section-title = t("loan.breakeven.charts_title")
+  .chart
+    h4.chart-title = t("loan.breakeven.revenue_by_product")
+    #breakeven-revenue-chart.chart-object
```

Comment:
If there is more than one breakeven table question, the chart only loads in the first spot, rather than the spot corresponding to the questions that have been answered. I've started work on fixing this, but that was not listed in the things that needed to be changed. Is this a priority?